README.md

```diff
@@ -65,138 +30,36 @@ After you're finished please remove all the comments and instructions!
 
 > The very first project of the 1st module of microverse full stack web development program
 
-**[your_project__name]** is a...
-
-## 🛠 Built With <a name="built-with"></a>
+**[first-microverse-project]** is a simple HTML project
```

Comment:
- [ ] Again, as my colleague opined, kindly consider removing the braces around your texts except it is a link (which will be immediately followed by a bracket) unlike this case. This allows readers to understand you and read your documentation with ease.